docker-compose.yml

```diff
@@ -40,27 +54,38 @@ services:
       - POSTGRE_HOST=db
       - LABEL_STUDIO_HOST=${LABEL_STUDIO_HOST:-}
       - JSON_LOG=1
-#      - LOG_LEVEL=DEBUG
+    #      - LOG_LEVEL=DEBUG
     volumes:
       - ./mydata:/label-studio/data:rw
+    networks:
```

Comment:
Let's move network to override file

Reply:
Having different networking approaches in the override and base files would make the networking only worse and harder to understand for users.
I would instead fall back to using the same networking structure you use in the base compose file -> everything in the default network as a free for all.

Reply:
ok, let's stick with existing approach